plugins/@grouparoo/hubspot/src/lib/export/options.ts

```diff
@@ -94,18 +104,21 @@ class OptionsHandler {
     const names = [];
     if (customObject?.properties) {
       for (const object of customObject.properties) {
-        if (object.name.startsWith("hs_")) {
+        if (
+          object.modificationMetadata.readOnlyValue === true ||
```

Comment:
above you are also skipping archived and calculated. Should we do that here too?